.gitignore

```diff
@@ -17,4 +17,7 @@ package-lock.json
 **/.history
 
 # Exception for extension recommendations
-!**/.vscode/extensions.json
\ No newline at end of file
+!**/.vscode/extensions.json
+
+# Config files for IDEA based IDE's
+.idea/
```

Comment:
undo

Reply:
I added it because it can be useful for jb users but ok

Reply:
for this one you can just create a .gitignore file in whatever folders you work in and then have the first line of it be ".gitignore" and then whatever you're ignoring so it doesn't commit.